### PR TITLE
rule update: add MITRE tags for rules

### PR DESCRIPTION
**Pull request by Kaizhe**
Added MITRE tags for default falco rules

### CODE REVIEW

**Comment by Kaizhe** [comment]
At the top level, we have the following categories are covered:

`Execution`, `Persistence`, `Privilege Escalation`, `Credential Access`, `Discovery`, `Lateral Movement`, `Exfiltration`, `Command and Control (in sub cat Port Knocking)`

https://attack.mitre.org/matrices/enterprise/linux/

**Comment by KnoxAnderson** [comment]
Maybe something like Mitre-Execution @Kaizhe

**Comment by Kaizhe** [comment]
👌

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -1107,7 +1107,7 @@
     Sensitive file opened for reading by trusted program after startup (user=%user.name
     command=%proc.cmdline parent=%proc.pname file=%fd.name parent=%proc.pname gparent=%proc.aname[2])
   priority: WARNING
-  tags: [filesystem]
+  tags: [filesystem, Credential Access]
```
Comment (mstemm) [inline rules/falco_rules.yaml]: I think this should be `mitre_credential_access` right?

**Pull request merged**